viam-cartographer.go

```diff
@@ -346,9 +347,19 @@ func (cartoSvc *cartographerService) GetSLAMProcessConfig() pexec.ProcessConfig
 	args = append(args, "-port="+cartoSvc.port)
 	args = append(args, "--aix-auto-update")
 
+	target := cartoSvc.executableName
+
+	appDir := os.Getenv("APPDIR")
+
+	if appDir != "" {
+		// The carto grpc server is expected to be in
+		// /usr/bin/ if we are running in an AppImage
+		target = appDir + "/usr/bin/" + strings.TrimPrefix(target, "/")
+	}
+
 	return pexec.ProcessConfig{
 		ID:      "slam_cartographer",
-		Name:    cartoSvc.executableName,
+		Name:    target,
```

Comment:
@kkufieta Do you not need `CMD` as @zaporter-work did here: https://github.com/viamrobotics/viam-orb-slam3/pull/38/files?

Reply:
Yes! Added.